cfpb/cfgov-refresh
# Regs3k: Navigate effective versions of regulations

This PR makes it possible to navigate versions in the Regulations 3000 UI and makes changes to the way effective dates are shown to users on the current law version of a regulation.

This PR will replace the use of the Text Introduction heading and links with the hard-coded use of the page's title and generated links to versions and search. So all of those should be removed from the Text Introduction block on a Regulation page otherwise you'll see double.

## Testing

1. Load past versions JSON or a database dump containing them (see chat), or create copies of effective versions with future or past effective dates.
2. Browse the regulation and navigate between versions using the "View all versions of this regulation" links.

## Screenshots

- **Current law**
   - Landing page: 
      ![image](https://user-images.githubusercontent.com/10562538/45370193-b24e8b00-b5b5-11e8-8ce7-ad8b7c7b55f7.png)
   - Section page: 
      ![image](https://user-images.githubusercontent.com/10562538/45370726-ce9ef780-b5b6-11e8-8854-11447d2e6cc7.png)

- **Past version**
   - Landing page: 
      ![image](https://user-images.githubusercontent.com/10562538/45370791-f55d2e00-b5b6-11e8-9f49-546664e985a8.png)
   - Section page: 
      ![image](https://user-images.githubusercontent.com/10562538/45370761-e5454e80-b5b6-11e8-8818-38cc23d4d2ca.png)


- **Future version**
   - Landing page: 
      ![image](https://user-images.githubusercontent.com/10562538/45370925-42d99b00-b5b7-11e8-9d39-00010b75eb1e.png)
   - Section page: 
      ![image](https://user-images.githubusercontent.com/10562538/45370944-51c04d80-b5b7-11e8-8df1-26e6236af68c.png)

- **All versions**
   ![image](https://user-images.githubusercontent.com/10562538/45370902-38b79c80-b5b7-11e8-8a77-052ca9adff86.png)



## Todos

- There is epic amounts of space on the landing page between the Wagtail and effective version-related bits. @contolini do you mind taking a look and seeing if there are better elements we can use there?
- Future effective versions *do not* show up on the main regulations listing landing page yet. That will be done in a future PR.

## Checklist
